go/cmd/dolt/commands/blame.go

```diff
@@ -543,8 +564,10 @@ func (bg *blameGraph) String(ctx context.Context, pkColNames []string) string {
 
 	t := pretty.NewWriter()
 	t.AppendHeader(header)
+	var p []rowMap
 	for _, v := range *bg {
 		pkVals := getPKStrs(ctx, v.Key)
+		pkVal := getPKVal(ctx, v.Key)
```

Comment:
Take a pass at simplifying this code here. You have two methods that are doing very similar things.